package.json

```diff
@@ -91,13 +91,13 @@
     "watch:css:components": "postcss \"src/components/**/*.css\" --dir \"dist/components\" --base \"src/components\" --watch",
     "watch:ts:components": "tsc --project . --watch",
     "watch": "run-p watch:**",
-    "start:dev": "yarn build && run-p watch storybook:start",
+    "start:dev": "npm-run-all build --parallel watch storybook:start",
```

Comment:
is this functionally the same, just with a different syntax?

Reply:
Yup. I didn't want to do the same thing in two different ways. I was also unsure how `&&` behaves on Windows.